velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -137,7 +137,29 @@ TEST_F(WindowTest, variableWidthAggregate) {
   })};
 
   testWindowFunction(input, "min(c2)", kOverClauses);
-  testWindowFunction(input, "max(c2)", kOverClauses);
+  testWindowFunction(input, "max(c2)", kOverClauses, {""}, false);
+}
+
+class DecimalAggregatesTest : public WindowTestBase {};
```

Comment:
We should avoid having multiple test classes in a single file. Ideally, each class is in a separate file and class name matches file name.

Also, let's move tests for Window operator into exec/tests.

Reply:
Updated, thanks for the feedback.